pkg/vms/Makefile.am

```diff
@@ -0,0 +1,90 @@
+SRCDIR=pkg/vms
+vmsdir = $(pkgdatadir)/vms
+
+vms_MODULES = \
+	$(SRCDIR)/_index.js \
+	$(SRCDIR)/index.js \
+	$(SRCDIR)/store.js \
+	$(SRCDIR)/actions.js \
+	$(SRCDIR)/reducers.js \
+	$(SRCDIR)/selectors.js \
+	$(SRCDIR)/middlewares.js \
+	$(SRCDIR)/services.js \
+	$(SRCDIR)/libvirt.js \
+	$(SRCDIR)/app.js \
+	$(SRCDIR)/hostvmslist.js \
+	$(SRCDIR)/vmdetail.js \
+	$(SRCDIR)/reloadSwitch.js \
+	$(SRCDIR)/helpers.js \
+	$(SRCDIR)/config.js \
+	$(NULL)
+
+vmsimagesdir = $(vmsdir)/images
+vmsimages_DATA = \
+        $(SRCDIR)/images/vm_shutdown.png \
```

Comment:
Is this file used? I would suggest not using an image here but an icon from the FontAwesome font + CSS colors .

http://fontawesome.io/icons/